README.md

```diff
@@ -1,12 +1,10 @@
-## News From The Northcoders API
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
-# nc-news-1
 
+# News From The Northcoders API
 
 Welcome to the Northcoders News API! This project serves as the backend API for Northcoders News, providing  the following endpoints: 
 
-## Overview Of Endpoints
-
+#  Overview Of Endpoints
+```
```

Comment:
These endpoints will be served on the /pi endpoint so it's not necessary to list them here